(2/2) Make TorchScript Preserve Fully Qualified Class Name for Python Exceptions: frontend change

shunting314 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #70471

Reland D33282878. This is the frontend change.

Differential Revision: [D33342569](https://our.internmc.facebook.com/intern/diff/D33342569/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D33342569/)!

shunting314 committed:
(2/2) Make TorchScript Preserve Fully Qualified Class Name for Python Exceptions: frontend change

Reland D33282878. This is the frontend change.

Differential Revision: [D33342569](https://our.internmc.facebook.com/intern/diff/D33342569/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D33342569/)!

[ghstack-poisoned]

gmagogsfm commented:
@iseeyuan We are planning to make a change that needs to coordinate with mobile runtime binary rollout, this is FC-breaking because non-default values are going to be actually used in about 2 weeks.

shunting314 committed:
Update on "(2/2) Make TorchScript Preserve Fully Qualified Class Name for Python Exceptions: frontend change"


Reland D33282878. This is the frontend change.

Differential Revision: [D33342569](https://our.internmc.facebook.com/intern/diff/D33342569/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D33342569/)!

[ghstack-poisoned]

shunting314 committed:
Update on "(2/2) Make TorchScript Preserve Fully Qualified Class Name for Python Exceptions: frontend change"


Reland D33282878. This is the frontend change.

Differential Revision: [D33342569](https://our.internmc.facebook.com/intern/diff/D33342569/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D33342569/)!

[ghstack-poisoned]
